app/api/dao/admin.py

```diff
@@ -2,9 +2,22 @@
 
 
 class AdminDAO:
+    """ Data Access Object for Admin functionalities.
```

Comment:
I'm sorry, now, I'm feeling picky. I told you about the typo, but I also mentioned about the extra space character you have between `"` and `Data`, The other doc string you have start with `"""Something` here you have a space, and I don't know why, is it on purpose? Can you fix that please? @sarnava1

Reply:
@isabelcosta done!